manifests/init.pp

```diff
@@ -171,6 +171,8 @@
 #
 # $dhcp_provider::              DHCP provider
 #
+# $dhcp_subnets::               DHCP subnets
```

Comment:
If you could also expand the description, that would be very useful - e.g. "List of subnets to restrict DHCP management to"